README.md

```diff
@@ -1,35 +1,171 @@
-# Blockbook docker
+# Blockbook Docker (Single Container)
 
-## Pre-requisites
-Should have [sysbox installed on your machine](https://github.com/nestybox/sysbox/blob/master/docs/developers-guide/build.md)
+This guide will help you build and run the Docker image for Blockbook Mainnet and Testnet, which is based on Ubuntu 22.04 and includes both backend and frontend components in a single container.
 
-## Main commands
-```
-sudo docker run -d --runtime=sysbox-runc -P -p <hostport>:9166 --name blockbook ranchimallfze/blockbook:1.0.0
-```
+## Prerequisites
 
-## Testing  
+-   [Docker installed on your machine](https://docs.docker.com/engine/install/).
+-   Clone or download the repository to your local machine.
 
-The code and steps required to run Docker version of Blockbook block explorer
+---
 
-```
-sudo docker build -t blockbook-althelper -f Dockerfile-althelper .
-sudo docker run -d --privileged -p 9166:9166 d5e56e218acd
-```
+# Mainnet
 
-```
-docker buildx create --driver-opt image=moby/buildkit:master --use --name insecure-builder --buildkitd-flags '--allow-insecure-entitlement security.insecure'
-docker buildx use insecure-builder
-docker buildx build --allow security.insecure ...(other build args)...
-sudo docker buildx build --allow security.insecure -t blockbook .
-docker buildx rm insecure-builder
+Steps to build the Docker image for mainnet.
 
+-   **Building the Docker Image**
 
-sudo docker run -v /var/run/docker.sock:/var/run/docker.sock -ti ubuntu:20.04
-```
+    Navigate to the directory where the repository is cloned or downloaded and build the Docker image:
 
-```
-sudo docker run -d --runtime=sysbox-runc -P -p 9167:9166 5018bee64419
+    ```sh
+    cd <path/to/cloned/repository>
+    docker build -t <imagename> .
+    ```
 
-sudo docker run -d --runtime=sysbox-runc --net=host -P 5018bee64419
-```
\ No newline at end of file
+-   **Running the Docker Container**
+
+    Create a named volume for persistent storage using:
+
+    ```sh
+    docker volume create <volume_name>
```

Comment:
At all the places where you have given a placeholder in the readme, also put examples there with real names. For things like volumes use default names so that people can just copy paste the commands and run.